docs/pages/components/file-uploader.md

```diff
@@ -0,0 +1,155 @@
+---
+title: File Uploader
+id: file-uploader
+keywords: file uploader,files
+sidebar: left-navigation-sidebar
+toc: false
+permalink: components/file-uploader.html
+folder: components
+summary:
+tags: [f3, a11y, theme]
+---
+
+The File Uploader component is an opinionated composition of the input and button components.
+Please see the `input` and `button` components documentation for the complete list of modifiers.
+<br/>
+<br/>
+`File Uploader` allows user to select the files when click on the input or browse button.
+Input element is readonly and populated with the selected file name.
+{: .docs-intro}
+<br/>
+
+## Cozy Mode
+
+{% capture default %}
+
+<div class="fd-form-item">
+    <div class="fd-file-uploader">
+        <input 
+        class="fd-input fd-file-uploader__input" 
+        onclick="browseFile('input1');" 
+        title="Select a file for uploading" 
+        aria-label="Select a file for uploading"
+        aria-live="polite"
+        autocomplete="off"
+        type="text" 
+        id="browse_input1" 
+        placeholder=" Choose a file for upload to localserver " 
+        tabindex="-1"
```

Comment:
Actually it seems fair, to add input with type of file, which will be hidden, like on [sapui5](https://openui5.hana.ondemand.com/entity/sap.ui.unified.FileUploader/sample/sap.ui.unified.sample.FileUploaderBasic) example

Reply:
iIt's already added input with type of file and its hidden as in SAP UI5.